kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/ResourceListTable.vue

```diff
@@ -3,19 +3,14 @@
   <core-table>
     <thead slot="thead">
       <tr>
-        <th class="core-table-icon-col">
+        <th class="core-table-main-col" colspan="3">
```

Comment:
Right here for example, the name header is right above the move arrows. Feels too far from the actual name column.

![localhost_8000_coach_ ipad](https://user-images.githubusercontent.com/7193975/38904652-73438ce4-4260-11e8-83ce-cc6f08447a0c.png)

Reply:
yeah, that's fair. okay I'll revert those changes